.github/workflows/sca_scan.yaml

```diff
@@ -0,0 +1,31 @@
+name: Snyk Software Composition Analysis Scan
+# This git workflow leverages Snyk actions to perform a Software Composition 
+# Analysis scan on our Opensource libraries upon Pull Requests to Main & 
+# Develop branches.  We use this as a control to prevent vulnerable packages 
+# from being introduced into the codebase. 
+on:
+  pull_request:
+    branches: 
+      - master
+      - develop
+jobs:
+  Snyk_SCA_Scan:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [16.x]
+    steps:
+      - uses: actions/checkout@master
```

Comment:
In the `static_analysis` action we use `@v2` instead of `@master`. I would prefer we reference versions wherever possible.

Reply:
Thanks, I've updated